apps/mobile-wallet/src/features/modals/BottomModalHeader.tsx

```diff
@@ -38,9 +40,10 @@ const BottomModalHeader = ({
             ))}
         </TitleContainer>
       </HeaderTitleOutterContainer>
-      {showCloseButton && (
+      {(showCloseButton || rightButton) && (
         <HeaderSideContainer align="right">
-          <CloseButton onPress={onClose} />
+          {rightButton}
```

Comment:
I'm realising that side buttons / components are misaligned. Not linked to this PR, but would be great to fix it :) 

<img width="124" height="169" alt="Image" src="https://github.com/user-attachments/assets/d79a10e1-4f53-465b-9cef-e5da0383f36c" />